docs/source/item-validation.rst

```diff
@@ -184,6 +212,36 @@ A `list` containing the location of the item schema. Could be a local path or a
 If you are working on a spider that produces multiple items types, you can define it
 as a `dict`:
 
+.. code-block:: python
+
+    # settings.py
+
+    SPIDERMON_VALIDATION_SCHEMAS: {
+        DummyItem: '/path/to/dummyitem_schema.json',
+        OtherItem: '/path/to/otheritem_schema.json',
+    }
+
+.. _SPIDERMON_VALIDATION_CERBERUS:
+
+SPIDERMON_VALIDATION_CERBERUS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default: ``None``
+
+A `list` containing the local path of the item schema.
+
+.. code-block:: python
+
+    # settings.py
+
+    SPIDERMON_VALIDATION_CERBERUS: [
+        '/path/to/schema.json',
+        'https://api.myjson.com/bins/gmdgl'
+    ]
+
+If you are working on a spider that produces multiple items types, you can define it
+as a `dict`:
```

Comment:
Give an example of this.

Reply:
Roger that. I will see if Quotes spider can have data scrapped going to 2 items which I remember it can. Thanks

Reply:
Looking good, now. 

![Screenshot_2019-08-15_17-52-33](https://user-images.githubusercontent.com/22801822/63094358-c3eaa080-bf85-11e9-9eec-d7e749be99ba.png)